config/default.json

```diff
@@ -0,0 +1,30 @@
+{
+    "server": {
+        "port": 3000,
+        "ip": "localhost",
+        "mongourl": "mongodb://localhost:27017/tf2pickup",
+        "auth": {
+            "required_hours": 1000,
+            "secret": "someamazingsecret",
+            "url": "/auth/steam"
```

Comment:
The auth URL doesn't need to be in one of the config files. As this will likely not change and we need to share this between client and server this can be just in the normal js config file.